docs/app/app.component.ts

```diff
@@ -89,8 +90,10 @@ export class AppComponent implements OnInit {
     constructor(private router: Router,
         private activatedRoute: ActivatedRoute,
         private navigation: NavigationService,
-        ngZone: NgZone) {
+        ngZone: NgZone,
+        siteThemeService: SiteThemeService) {
         (<any>window).ngZone = ngZone;
+        siteThemeService.load();
```

Comment:
We don't actually need to load the service in the open source site as far as I'm aware. The service just needs to be here because all the playground code is here too. The MF site has it's own independent AppComponent.